.github/workflows/test.yml

```diff
@@ -0,0 +1,16 @@
+on:
+  push:
+    branches:
+      - main
```

Comment:
What does "main" refer to here?  Is it an alias for the Default branch configured for a repo?  (I'm just curious.)

Reply:
Ugh yes this is me not paying attention while trying to adapt examples, thanks for pointing this out.

Reply:
Changed to `debian/master`